ChangeLog.d/fix_test_memory_leak.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix memory leak that occured in error case in build_transforms() in test_suite_ssl.function
```

Comment:
No need for a changelog entry when fixing a bug in test code.

Reply:
Ok, I wasn't sure, so erred on the side of caution